migrations/versions/dd7968d8d413_.py

```diff
@@ -0,0 +1,36 @@
+"""empty message
+
+Revision ID: dd7968d8d413
+Revises: 194a5a2a44ef
+Create Date: 2018-06-02 21:57:00.799206
+
+"""
+
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = 'dd7968d8d413'
+down_revision = '194a5a2a44ef'
```

Comment:
Just update this `down_revision` to `49f3a33f5437` instead of adding another migration version which only merges two heads.